ygen/gogen.go

```diff
@@ -46,6 +46,12 @@ const (
 	// DefaultGoyangImportPath is the default path for the goyang/pkg/yang library that
 	// is used in the generated code.
 	DefaultGoyangImportPath string = "github.com/openconfig/goyang/pkg/yang"
+	// defaultMetadataPrefix is the default string that is used to prefix the name
+	// of metadata fields in the output Go structs.
+	defaultAnnotationPrefix string = "Λ"
```

Comment:
Ah we even have such constant - so if we make this public, we could use it in the flag, instead of having the literal in multiple locations.

Reply:
Done :-)